extras/exporters/gltf-exporter.js

```diff
@@ -120,7 +120,19 @@ const textureSemantics = [
     'emissiveMap'
 ];
 
+/**
+ * Implementation of the GLTF 2.0 format exporter.
+ */
 class GltfExporter extends CoreExporter {
+    /**
+     * Creates an instance of the GltfExporter.
+     * @function
+     */
+    // No actual constructor code here, just JSDoc.
```

Comment:
TBH, I think this is unnecessary (I'm not even convinced this is doing to the right thing anyway). If the superclass' constructor has something like `Create a new instance of the exporter.`, I think that's sufficient to be shown in the subclass too.